QUIZ 1 program.py

```diff
@@ -0,0 +1,32 @@
+
+# # 7.  Write a program to create "intro.txt" file.  Print only even number of row.
+
+# # WRITING 
+from csv import writer
+
+info=[["My name is Roshni Singh."],
+["I am 18 years old"],
+["I live in surat."],
+["I Have completed 12th."],
+["My hobby is Drawing,travelling,Dancing"],
+["I study in VTCBCSR College."],
+["I want to become a Software engineer."],
+["Currently i am doing BCA."],
+["My Favourite colour is Black."],
+["My dream place is KEDARNATH."]]
```

Comment:
I want to take multiple line input means user input 10 lines in one input() and store in variable. is it possible? if yes...please show me your code.

Reply:
**YES,** we can take 10 input from user.
**HERE is my code,
line=[]
while true:
             line=input()
             if line:
                      lines.append(line+"\n")
             else:
                      break
text="\n".join(lines)**